emcc.py

```diff
@@ -181,6 +181,10 @@ def run():
     print shared.get_llvm_target()
     exit(0)
 
+  elif '--cflags' in sys.argv:
```

Comment:
So, general question: what other emscripten flags/options are there that affect the clang flags? Does this output include the effect of those flags?